recipes/nbclassic/meta.yaml

```diff
@@ -0,0 +1,49 @@
+{% set name = "nbclassic" %}
+{% set version = "0.2.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: fc45b48045c217de3df1f942ce8548a9aa81cbc78836958e2bb829aa49beb4d7
+
+build:
+  number: 0
+  noarch: python
+  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed --no-cache-dir -vvv"
+  entry_points:
+    - jupyter-nbclassic = nbclassic.notebookapp:main
+
+requirements:
+  host:
+    - python >=3.6
+    - pip
+    - setuptools
+  run:
+    - python >=3.6
+    - jupyter_server >=1.0.1,<2.0.0
+    - notebook <7
+
+test:
+  imports:
+    - nbclassic
+
+about:
+  home: http://github.com/jupyterlab/nbclassic
+  license: BSD 3-Clause
+  license_file: LICENSE
+  summary: Jupyter Notebook as a Jupyter Server Extension.
+
+  description: |
+    Jupyter Notebook as a Jupyter Server Extension.
+  dev_url: https://github.com/jupyterlab/nbclassic
+
+extra:
+  recipe-maintainers:
+    - Zsailer
+    - jasongrout
+    - blink1073
+    - afshin
```

Comment:
cc @Zailer @jasongrout @blink1073 @afshin

Let me know if it's fine to add you as maintainers.

Reply:
Sure, thanks!

Reply:
Cheers, Jeremy!